### ISSUE TITLE
Allow selection in Strip Plots

### ISSUE THREAD

**Issue by dg-datavisyn** [datavisyn/tdp_core#639]
* Release number or git hash: Version: 10.0.1-SNAPSHOT
* Web browser version and OS: Google Chrome Version 98.0.4758.102 (Offizieller Build) (64-Bit) on Win 11
* Environment (local or deployed): https://ordino.dev.app.datavisyn.io/

### Steps to reproduce

1. Open a tissue data ranking
2. Open the Vis panel
3. Create a Strip Plot

### Observed behavior
* I can't select anything

### Expected behavior
* I want to select dots just like in the scatter plots with the same behavior

**Comment by dg-datavisyn** [comment]
I agree with you. The colors are not needed since the attribute is already expressed by a different channel. The question is: Do we want to use gray as the default color for our visualizations? Most users associate gray elements with disabled elements.

**Comment by alexsb** [comment]
I think the association of gray to disabled is context dependent: if some marks are colored and others are light-gray, it looks like it's filtered out or disabled. I don't think that's really a concern here though. 

But I do think there are some aesthetic choices that matter: things shouldn't be boring. We could choose a color as our default mark color, e.g., make all of them blue, and only use color by category if explicitly specified – we should look at this in the context of the whole app. We'd have to experiment how this interacts with selections, I agree.
